client/evilwallet/evilwallet.go

```diff
@@ -191,7 +193,7 @@ func (e *EvilWallet) requestFaucetFunds(wallet *Wallet) (outputID utxo.OutputID,
 	if err = RateSetterSleep(clt, true); err != nil {
 		return
 	}
-	err = clt.BroadcastFaucetRequest(addr.Base58())
+	err = clt.BroadcastFaucetRequest(addr.Base58(), 12)
```

Comment:
This is the faucet request PoW requirement right? Better maybe put it in a const

Reply:
Yeah, that's a temporary fix. Ideally it should be put into a parameter and passed from outside, but we wanted to have something quick. We can add it as a parameter in a different PR.